pkg/config/config.go

```diff
@@ -146,6 +146,8 @@ func init() {
 	Datadog.SetDefault("proc_root", "/proc")
 	Datadog.SetDefault("histogram_aggregates", []string{"max", "median", "avg", "count"})
 	Datadog.SetDefault("histogram_percentiles", []string{"0.95"})
+	Datadog.SetDefault("histogram_copy_to_distribution", false)
```

Comment:
You could simply use `BindEnvAndSetDefault` instead of `SetDefault` +  `BindEnv`.